[Docs] Update references in Ray Docs about batch inference (#36968)
ray-project/ray: doc/source/serve/tutorials/batch.md

- How to configure the batch size.
- How to query the model in Python.

> This tutorial should help when you want to serve online queries and your model can take advantage of batching. For example, linear regressions and neural networks use CPU and GPU's vectorized instructions to perform computation in parallel. Performing inference with batching can increase the *throughput* of the model as well as *utilization* of the hardware.

Reviewer: ```suggestion
This tutorial is a guide for serving online queries when your model can take advantage of batching. For example, linear regressions and neural networks use CPU and GPU's vectorized instructions to perform computation in parallel. Performing inference with batching can increase the *throughput* of the model as well as *utilization* of the hardware.
```

> For _offline_ batch inference with large datasets, see [batch inference with Ray Data](batch_inference_home) instead.

Reviewer: ```suggestion
For _offline_ batch inference with large datasets, see [batch inference with Ray Data](batch_inference_home).
```
